signin.html

```diff
@@ -52,5 +52,6 @@
         </div>
       </div>
     </div>
+    <script src="script\sign.js"></script>

```

Comment:
제가 멘토링에서 말씀드렸던 module script 를 한번 적용해보세요!

```suggestion
    <script type='module' src="script\sign.js"></script>

```

Reply:
[894935c](https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/pull/270/commits/894935c5fbf52615b3cfc14bd6fadf414fd13a1c) 완료했습니다~!~